images/full-history/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM developmentseed/osmseed-osm-processor:0.1.0-n487.hbce9469
+FROM developmentseed/osmseed-osm-processor:v2
```

Comment:
Should this be changed?

Reply:
yes, it is to include the Azure CLI in the containers